src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/convert/MappingDocumentDbConverter.java

```diff
@@ -81,43 +82,37 @@ public MappingDocumentDbConverter(
 
     @Override
     public void write(Object sourceEntity, Document document) {
-        if (sourceEntity == null) {
-            return;
-        }
-
-        final DocumentDbPersistentEntity<?> entity = mappingContext.getPersistentEntity(sourceEntity.getClass());
-        writeInternal(sourceEntity, document, entity);
+        // Not used anymore
+        throw new NotImplementedException();
     }
 
-    public void writeInternal(final Object entity,
-                              final Document targetDocument,
-                              final DocumentDbPersistentEntity<?> persistentEntity) {
-        if (entity == null) {
-            return;
+    public Document writeDoc(Object sourceEntity) {
+        if (sourceEntity == null) {
+            return null;
         }
 
-        if (persistentEntity == null) {
-            throw new MappingException("no mapping metadata for entity type: " + entity.getClass().getName());
-        }
+        try {
+            final DocumentDbPersistentEntity<?> persistentEntity =
+                    mappingContext.getPersistentEntity(sourceEntity.getClass());
```

Comment:
Empty line?

Reply:
Or use temp local variable

Reply:
What does empty line and temp local variable means here? Did not catch.

Reply:
Whitespaces after `=` is because we have a 120 characters per line limitation, so it's separated into 2 lines.